.circleci/config.yml

```diff
@@ -1,42 +1,38 @@
-# Java Maven CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-java/ for more details
-#
 version: 2
-executorType: machine
 jobs:
   build:
-    docker:
-      # specify the version you desire here
-      - image: circleci/openjdk:11-jdk
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
-
-    working_directory: ~/repo
-
-    environment:
-      # Customize the JVM maximum heap limit
-      MAVEN_OPTS: -Xmx3200m
-
+    machine:
+      image: ubuntu-1604:202004-01
```

Comment:
16.04? It doesn't really matter, but why so back-versioned?

Reply:
didn't know where to find newer versions and this was the latest I could find from the CI page 🤷‍♂️

Reply:
I think you can use any of these https://hub.docker.com/_/ubuntu/ but idk

Reply:
cool; good to know